### PR TITLE
Adding Linter for .js and .html-termplates

### PR DESCRIPTION
**Pull request by BenediktMKuehne**
using djlint(pip) & jscheck(npm)

### CODE REVIEW

File: check_project.sh [hunk 1]
```diff
@@ -10,15 +10,64 @@
...
 
+RED='\033[0;31m'
 GREEN='\033[0;32m'
 ORANGE='\033[0;33m'
 BOLD='\033[1m'
 NC='\033[0m' # no color
 
+# checks js-scripts with jshint for errors
+# config @ ./embark/static/.jshintrc
+jscheck(){
```
Comment (m-1-k-3) [inline check_project.sh:25]: could we include a quick check if jshint is already available. If it is not available give a note that the installer script should be used

File: check_project.sh [hunk 2]
```diff
@@ -10,15 +10,64 @@
...
+  done
+}
+
+# uses djlint to check for errors in all html-template files inside /embark (Django-root-dir)
+# no config
+templatechecker(){
+  mapfile -t HTML_FILE < <(find embark -iname "*.html")
+  for HTML_FILE in "${HTML_FILE[@]}"; do
+    echo -e "\\n""$GREEN""Run tidy on $HTML_FILE:""$NC""\\n"
+    python3 -m djlint "$HTML_FILE"
```
Comment (m-1-k-3) [inline check_project.sh:57]: same as with jshint

Comment (BenediktMKuehne) [reply check_project.sh:57]: image

would be in the next pull-request

**Review by m-1-k-3 — APPROVED** [review]
huge PR. Thanks for the contribution

**Pull request merged**